src/UnfoldMakie.jl

```diff
@@ -95,8 +95,10 @@ export plot_channelimage
 export plot_channelimage!
 
 export to_positions
-export config_kwargs!
 export eeg_array_to_dataframe
 export eeg_topoplot_series
 export nonnumeric # reexport from AoG
+
+export config_kwargs!
```

Comment:
you marked the last comment as resolved; but you still export config_kwargs! - I think it shouldnt be exported. whats hte argument?

Reply:
I can delete it, just wanted to be able to see docstrings of it